doc/manuals/contribution_guidelines.md

```diff
@@ -405,6 +405,17 @@ directly or indirectly are forced to have that construct as well.
 
 *How to check*: using an intelligent ```grep```.
 
+#### M12bis (`using namespace` in source code files)
+
+*Rule 1*: the construct ```using namespace XXX``` must not be used in source code files.
+*Rule 2*: the construct ```using XXX::YYY" **may** be used in source code files, but even better is to **not** use `using` but to write out the namespace (call it prefix if you want) wherever needed.
```

Comment:
Subjective ... perhaps a little. I don't think anybody could oppose to what I say though. It *is* clearer to write out the namespace everywhere. That's more than an opinion (in my opinion :-D)

The rule 2 actually says 'using XXX::YYY" **may** be used', leaving it open to the programmer to decide.

That said, I'll remove rule 2 to not spend any more time on this